LiteCore/RevTrees/RevTree.cc

```diff
@@ -606,6 +606,28 @@ void RevTree::resetConflictSequence(const Rev* winningRev) {
         return false;
     }
 
+    bool RevTree::ensureConflictStateConsistent() {
+        if (hasConflict()) {
+            sort();
+            bool recovered = false;
+            // _revs[0] is current rev after sort
+            for (auto rev = _revs.begin() + 1; rev != _revs.end(); rev++) {
+                auto f = (*rev)->flags;
+                // a undeleted, non-closed, unconflict leaf rev
+                if ((f & Rev::kLeaf) &&
+                    !(f & Rev::kClosed) &&
+                    !(f & Rev::kDeleted) &&
+                    !(f & Rev::kIsConflict)) {
+                        (*rev)->addFlag(Rev::kIsConflict);
+                        recovered = true;
```

Comment:
Add logging here for debugging

Reply:
Added some logging code, and it does work in android emulator.